### ISSUE TITLE
Initial four output channels should remain fixed

### ISSUE THREAD

**Issue by Boroal** [FrSkyRC/ETHOS-Feedback-Community#286]
my request when setting up a model the first four channels remain the default order and any additional channels start at 5.
I think a couple of people have tried to express this differently. I thought I would have a go as I have only set up two models and it is already driving me mad. As you are aware you set the default channel order for example; aileron, elevator, motor, rudder; to the first four channels. It is then standard practice to plug the relevant servos/motor into these channels. In this case the motor is always channel 3. This is a good safety feature as you are less likely to confuse channels when it comes to mixing, reversing etc.
The problem comes when you set up a model with more than one channel for any of the first four. The most common issue is a second aileron channel.  The wizard allocates the extra channel to the next number, in this case channel 2. Therefore bumping the next three so elevator becomes 3, motor 4 and rudder 5. Rather than changing the first four channels the second aileron channel should be 5.

**Comment by aalain8** [comment]
Cela me pose aussi des problèmes et je suis d'accord avec le message précédent. 
Tout le monde serait satisfait si on pouvait choisir l'ordre des voies à la fin de la création du modèle.
Merci.
Alain

**Comment by jfint** [comment]
I wish to vote down this request, this would significantly limit the system in an undesirable way.

**Comment by MrD-RC** [comment]
I completely agree with this. It annoys me too. I think people (@jreis1 @aalain8 and @jfint ) are getting confused with the term “fixing the channel order”. Of course, if you want to move channels, you’re completely able to. That should not change.

What @Boroal is referring to is the new model wizard abiding by the chosen channel order on the transmitter. So if you create a new model of a standard plane with 2 ailerons, the wizard currently sets them as channels 1 & 2, which messes up the AETR channel order. The wizard should set the ailerons to channels 1 & 5. This keeps AETR intact. This is certainly what I would expect to happen, and it annoys the hell out of me that I have to move 4 channels around to put this right. Likewise, if someone changes the default channel order to TAER, the ailerons should be on channels 2 & 5. 

If you wanted your ailerons on channel 13, rudder on channel 24, and elevator on channel 2. You would still be able to do this in the mixer. This issue is about the order that is created by the new model wizard only.

**Comment by jreis1** [comment]
> I completely agree with this. It annoys me too. I think people (@jreis1 @aalain8 and @jfint ) are getting confused with the term “fixing the channel order”. Of course, if you want to move channels, you’re completely able to. That should not change.
> 
> What @Boroal is referring to is the new model wizard abiding by the chosen channel order on the transmitter. So if you create a new model of a standard plane with 2 ailerons, the wizard currently sets them as channels 1 & 2, which messes up the AETR channel order. The wizard should set the ailerons to channels 1 & 5. This keeps AETR intact. This is certainly what I would expect to happen, and it annoys the hell out of me that I have to move 4 channels around to put this right. Likewise, if someone changes the default channel order to TAER, the ailerons should be on channels 2 & 5.
> 
> If you wanted your ailerons on channel 13, rudder on channel 24, and elevator on channel 2. You would still be able to do this in the mixer. This issue is about the order that is created by the new model wizard only. It wouldn’t limit anything. It would just make the channel order created by the wizard what most people would expect.

Hey 

I was referring to something like this, where you can select the channel of each surface when doing the wizard. It serves both purposes ;)

![814899B7-4099-4196-80F1-82DFDBCCD6FF](https://user-images.githubusercontent.com/25162166/119195625-e07e0b80-ba7c-11eb-8549-60cea9bfb71d.jpeg)

**Comment by MrD-RC** [comment]
Sorry @jreis1 you’re right.

That may help. But it may also take away from the simplicity aspect that the wizard currently has.

**Comment by aalain8** [comment]
-The SX8-6 and SR8 pro receivers do not match the default Ethos choice!
-Sometimes on a F5J or F3K with a GR8 the servos are connected in an illogical way! Place and length of the cables.
-On RCG this topic had raised a lot of comments in favour of this modification.
From my point of view, at the end of the model creation wizard, adding a page that lists the order of the channels with the possibility to modify them would be perfect.
It's a bit of a cosmetic request but I think it's important, as I plan to continue with Ethos.
Alain

**Comment by MrD-RC** [comment]
I like the idea of a confirmation screen for channel numbers @aalain8

**Comment by IJK-CZ** [comment]
Agree, the best (and not complicated solution) would be an option to set /confirm channel numbers when model is defined. I also preffer AETR almost everywhere, but it should not be fixed. But current Ethos stage is not good - you cannot simple exchange channel outputs, there is always need to use empty channels for move to prevent channel names mess / confusion. It is not user friendly.

**Comment by Hobby4life** [comment]
I agree witj IJK-CZ, AETR is more likely the standard nowadays. Even most flight controllers expect AETR configuration (via SBUS)
The wizard is fine, only needs the ability to change the channel outputs too.

**Comment by keiner0815** [comment]
I mentioned the same idea in more detail in this Issue, but it has been closed:
https://github.com/FrSkyRC/ETHOS-Feedback-Community/issues/94

It is just a possibility to have the channels in a fixed order from the wizard.
This list should be changeable, because everyone has his own channel order. But once set, it will remain the same in the following models created with the wizard.

And **of course** it is still possible to change the channels afterwards.
But, as mentioned in the issue above, I see it extremely dangerous to switch channel orders dynamically, just by changing anything in the settings (number of aileron servos for example). In this case the channel for the motor gets moved to another channel without giving even a warning!!!

**Comment by janekx** [comment]
AETR xxx should be thye way we go

**Comment by pstasek** [comment]
I don't think there should be any fixed order, the better option would be if the wizard allowed to assign channel numbers according to the numbers when setting up the model. Also, a button that would allow copying the name of the mix to channel on demand would be nice. Just a cosmetic issue though.

**Comment by Hobby4life** [comment]
> I don't think there should be any fixed order, the better option would be if the wizard allowed to assign channel numbers according to the numbers when setting up the model. Also, a button that would allow copying the name of the mix to channel on demand would be nice. Just a cosmetic issue though.

That is the best way to go, I vote for that too. Otherwise AETR is the best secundary option.
Most FBL and other flight controllers rely on the AETR order.

**Comment by bsongis-frsky** [comment]
![image](https://user-images.githubusercontent.com/50834778/123073880-1ebd7080-d417-11eb-94f1-b40036e35b1f.png)

**Comment by bsongis-frsky** [comment]
![image](https://user-images.githubusercontent.com/50834778/123073947-2ed55000-d417-11eb-9c7d-421a9d5a7f49.png)
